java/src/jmri/jmrix/loconet/messageinterp/MessageInterpretation_cs.properties

```diff
@@ -520,6 +520,48 @@ LN_MSG_OPC_MULTI_SENSE_TRANSP_ZONEH = H
 # replaced by a decimal representation of the LS 4 bits of byte 2 of the message.
 LN_MSG_OPC_MULTI_SENSE_TRANSP_ZONE_UNKNOWN = (Nezn\u00e1m\u00fd {0})
 
+# Note: in # LN_MSG_OPC_MULTI_TRANSP_RAILCOM_REPORT, the following parameter
+# substitutions will be made:
+#   {0} is a string representing the decoder address which triggered the report,
+#       and whether that address is 'long' or 'short', from
+#       LN_MSG_LOCO_ADDRESS_HELPER_MIXED_ADDRESS or
+#       LN_MSG_LOCO_ADDRESS_HELPER_UNMIXED_ADDRESS, as appropriate
+#   {1} is a number representing the RailCom App:Dyn Index value
+#   {2} is a number representing the RailCom Dynamic Value
+LN_MSG_OPC_MULTI_SENSE_TRANSP_RAILCOM_REPORT = Transponder address {0} reporting {1}.\n
+
+# Note: in # LN_MSG_RAILCOM_REPORT, the following parameter
+# substitutions will be made:
+#   {0} is a number representing the RailCom App:Dyn Index value
+#   {2} is a string representing the RailCom App:Dyn Index value
+#   {3} is a number representing the RailCom Dynamic Value
+LN_MSG_RAILCOM_REPORT = RailCom App:Dyn Index {0} {1}with a value of {2}
+
+# Note: in # LN_MSG_OPC_MULTI_SENSE_LONG_TRANSP_REPORT, the following parameter
+# substitutions will be made:
+#   {0} is a string representing the decoder address which triggered the report,
+#       and whether that address is 'long' or 'short', from
+#       LN_MSG_LOCO_ADDRESS_HELPER_MIXED_ADDRESS or
+#       LN_MSG_LOCO_ADDRESS_HELPER_UNMIXED_ADDRESS, as appropriate
+#   {1} is a string representing the direction headed
+#   {2} is a string representing the new state of the decoder in the zone, from
+#       LN_MSG_OPC_MULTI_SENSE_TRANSP_HELPER_IS_PRESENT or LN_MSG_OPC_MULTI_SENSE_TRANSP_HELPER_IS_ABSENT,
+#       as appropriate.
+#   {3} is a string representing the system name of the reporter associated with the
+#       transponding report, if available.
+#   {4} is a string representing the user name of the reporter associated with the
+#       transponding report, if available.
+LN_MSG_OPC_MULTI_SENSE_LONG_TRANSP_REPORT = Transponder address {0} facing {1} {2} {3} ({4}).\n
+LN_MSG_DIRECTION_EAST = East
+LN_MSG_DIRECTION_WEST = West
```

Comment:
Same as for .properties

Reply:
Added usage comments and included the words _HELPER for this and LN_MSG_DIRECTION_WEST